rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
+  Smarts, Mol, Smiles = range(3)
```

Comment:
Sorry to cut in, but this looks to be a good time for [`enum`](https://docs.python.org/3/library/enum.html):

```python
    from enum import Enum
   
     # ...
    
    InputFormat = Enum('InputFormat', ['SMARTS', 'MOL', 'SMILES'])
```